tests/phpunit/api/v3/PaymentTest.php

```diff
@@ -882,6 +882,75 @@ public function testCreatePaymentPayLater(): void {
     $this->validateAllPayments();
   }
 
+  /**
+   * Test create payment api for pay later contribution
+   *
+   * @throws \CRM_Core_Exception
+   */
+  public function testCreatePaymentMultipleTrxnID(): void {
+    $this->createLoggedInUser();
+    $processorID  = $this->paymentProcessorCreate();
+    $contributionParams = [
+      'total_amount' => 100,
+      'currency' => 'USD',
+      'contact_id' => $this->individualCreate(),
+      'financial_type_id' => 1,
+      'contribution_status_id' => 2,
+      'is_pay_later' => 1,
+      'trxn_id' => 'trxn_1'
+    ];
+    $contribution = $this->callAPISuccess('Contribution', 'create', $contributionParams);
+    //add payment for pay later transaction
```

Comment:
Also non-blocking but the drupal standard (that we don't enforce) is a space & then capital letter & full stop at the end I think